src/app/users/users.component.html

```diff
@@ -95,4 +95,4 @@ <h1 i18n>All Users</h1>
   </mat-table>
   <mat-paginator #paginator [pageSize]="10" [pageSizeOptions]="[5, 10, 20]"></mat-paginator>
 </div>
-<div class="view-container">{{message}}</div>
+<div class="view-container" *ngIf="!isUserAdmin">{{message}}</div>

```

Comment:
I see the whitespace out there but it would have been better to change the style of message section or instead of view-container making it something else.